packages/tokens/src/rwa/claim_issuer/storage.rs

```diff
@@ -529,24 +532,28 @@ pub fn remove_key(e: &Env, public_key: &Bytes, registry: &Address, scheme: u32,
     let signing_key = SigningKey { public_key: public_key.clone(), scheme };
 
     // Remove registry from Registries mapping
-    let registries_key = ClaimIssuerStorageKey::Registries(signing_key.clone());
-    let mut registries: Vec<Address> = e
+    let regsitries_storage_key = ClaimIssuerStorageKey::Registries(signing_key.clone());
+    let mut registries: Vec<(u32, Address)> = e
         .storage()
         .persistent()
-        .get(&registries_key)
+        .get(&regsitries_storage_key)
         .unwrap_or_else(|| panic_with_error!(e, ClaimIssuerError::KeyNotFound));
 
     // Find and remove the specific registry
```

Comment:
again, not the registry but the specific pair, all the comments should be refactored